src/sync/index.js

```diff
@@ -3,7 +3,10 @@ const _ = require('lodash');
 const connectDB = require('../db')
 
 const Qweb3 = require('qweb3');
-const qclient = new Qweb3('http://bodhi:bodhi@localhost:13889');
+const Qweb3Contract = require('qweb3/src/contract');
+const QTUM_RPC_ADDRESS = 'http://bodhi:bodhi@localhost:13889';
+const qclient = new Qweb3(QTUM_RPC_ADDRESS);
+
```

Comment:
Put QTUM_RPC_ADDRESS in a config file instead of sync/index.js like bodhi-api did?

Reply:
done